addons/hr_recruitment_survey/security/hr_recruitment_survey_security.xml

```diff
@@ -1,188 +0,0 @@
-        <record id="survey_question_rule_recruitment_manager" model="ir.rule">
-            <field name="name">Survey question: recruitment manager: all recruitment</field>
-            <field name="model_id" ref="survey.model_survey_question"/>
-            <field name="domain_force">[('survey_id.survey_type', '=', 'recruitment')]</field>
-            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_manager'))]"/>
-            <field name="perm_unlink" eval="1"/>
-            <field name="perm_write" eval="1"/>
-            <field name="perm_read" eval="1"/>
-            <field name="perm_create" eval="1"/>
-        </record>
-        <record id="survey_question_answer_rule_recruitment_manager" model="ir.rule">
-            <field name="name">Survey question answer: recruitment manager: all recruitment</field>
-            <field name="model_id" ref="survey.model_survey_question_answer"/>
-            <field name="domain_force">['|', ('question_id.survey_id.survey_type', '=', 'recruitment'),
-                ('matrix_question_id.survey_id.survey_type', '=', 'recruitment')]</field>
-            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_manager'))]"/>
-            <field name="perm_unlink" eval="1"/>
-            <field name="perm_write" eval="1"/>
-            <field name="perm_read" eval="1"/>
-            <field name="perm_create" eval="1"/>
-        </record>
-        <record id="survey_invite_recruitment_manager" model="ir.rule">
-            <field name="name">Survey invite: recruitment manager: all recruitment</field>
-            <field name="model_id" ref="survey.model_survey_invite"/>
-            <field name="domain_force">[('survey_id.survey_type', '=', 'recruitment')]</field>
-            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_manager'))]"/>
-            <field name="perm_unlink" eval="0"/>
-            <field name="perm_write" eval="1"/>
-            <field name="perm_read" eval="1"/>
-            <field name="perm_create" eval="1"/>
-        </record>
-
-        <!--special rights for recruitment officer on recruitment surveys-->
-        <record id="survey_user_input_rule_recruitment_user" model="ir.rule">
-            <field name="name">Survey user input: recruitment officer: unrestricted or in restricted users</field>
-            <field name="model_id" ref="survey.model_survey_user_input"/>
-            <field name="domain_force">[
-                '&amp;', ('survey_id.survey_type', '=', 'recruitment'),
-                '|',  ('survey_id.restrict_user_ids', 'in', user.id),
-                        ('survey_id.restrict_user_ids', '=', False)]</field>
-            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
-            <field name="perm_unlink" eval="0"/>
-            <field name="perm_write" eval="0"/>
-            <field name="perm_read" eval="1"/>
-            <field name="perm_create" eval="0"/>
```

Comment:
I see that beside the managers, the users and interviewers are restricted to read operations for the three `survey`, `survey_input` and `survey_line` models. I fail to see where this is implemented now.